Template10 (Library)/Controls/HamburgerButtonInfo.cs

```diff
@@ -92,7 +92,14 @@ public double MaxWidth
             set { Set(ref _MaxWidth, value); }
         }
 
-        public override string ToString() => string.Format("{0}({1})", PageType?.ToString() ?? "null", PageParameter?.ToString() ?? "null");
+		string _navigationGroup;
```

Comment:
Sorry to come across as tedious, but why are all these extra spaces added? Is it just an issue with github's Diff Viewer?